go/vt/vtctl/workflow/server.go

```diff
@@ -1069,10 +1138,15 @@ func (s *Server) getWorkflowCopyStates(ctx context.Context, tablet *topo.TabletI
 
 	copyStates := make([]*vtctldatapb.Workflow_Stream_CopyState, len(result.Rows))
 	for i, row := range result.Rows {
-		// These fields are technically varbinary, but this is close enough.
+		streamId, err := row[0].ToCastInt64()
+		if err != nil {
+			return nil, fmt.Errorf("failed to cast vrepl_id to int64: %v", err)
+		}
```

Comment:
This comment was about the table name and lastpk. There's no reason to case the vrepl_id field: https://github.com/vitessio/vitess/blob/main/go/vt/sidecardb/schema/vreplication/copy_state.sql#L20